index.bs

```diff
@@ -10229,6 +10253,8 @@ Constructors</h5>
 
 			1. Set <var>node</var>'s <a>processor reference</a> to
 				<var>processor</var>.
+
+			1. Return <var>processor</var>.
```

Comment:
This step is not needed or doable, since the caller of this algorithm (the IDL steps for constructors) already has the object and isn't expecting these steps to return anything.